compute_mutation_probabilities.py

```diff
@@ -119,7 +138,24 @@ def run( args ):
         for p, obs in patientToObserved.iteritems():
             assert( np.abs(P[:, patientToIndex[p]-1].sum() - obs) < 0.1)
 
-        # Add pseudocounts to entries with no mutations observed
+        # Construct mutation matrix to compute marginals
+        A = np.zeros(np.shape(P), dtype=np.int)
+        for i, j in edge_list:
+            A[i-1, j-1] = 1
+        r = np.sum(A, 1)
+        s = np.sum(A, 0)
+
+        # Post-process weight matrix to assign same weight to entries with same marginals
+        P = postprocess_weight_matrix(P, r, s)
+
+        # Verify the weights again
+        for g, obs in geneToObserved.iteritems():
+            assert( np.abs(P[geneToIndex[g]-1].sum() - obs) < 0.1)
+
+        for p, obs in patientToObserved.iteritems():
+            assert( np.abs(P[:, patientToIndex[p]-1].sum() - obs) < 0.1)
```

Comment:
I was hoping that you knew!  I changed it to something more conservative, given that the numerical error will be bounded by the number of floating point operations.